packages/utils/typescript/lib/generators/schemas/index.js

```diff
@@ -34,7 +34,10 @@ const emitDefinitions = (definitions) => {
     ts.ScriptKind.TS
   );
 
-  const printer = ts.createPrinter({ newLine: true, omitTrailingSemicolon: true });
+  const printer = ts.createPrinter({
+    newLine: ts.NewLineKind.LineFeed,
```

Comment:
I'm wondering if we should check the os before deciding what NewLineKind we need 🤔

Reply:
I think the best, for now, would be to use the newLine set by default by TS

https://github.com/microsoft/TypeScript/blob/8749fb5c0a4a0e3407eb099322c85b850cc642aa/src/compiler/emitter.ts#L1384

Reply:
Done